### PR TITLE
JBPM-4664:advanced and baisc mode for process instance include: new details…

### PR DESCRIPTION
**Pull request by bxf12315**
…page and  create module jbpm-console-ng-generic-preferences

@Salaboy 
Please review it!

Thanks

### CODE REVIEW

File: jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/ProcessInstanceDetailsWholeViewImpl.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HTMLPanel;
+import com.google.gwt.user.client.ui.Widget;
+
+@Dependent
+public class ProcessInstanceDetailsWholeViewImpl extends Composite implements ProcessInstanceDetailsPresenter.ProcessInstanceDetailsWholeViewUpdate {
+
```
Comment (Salaboy) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/ProcessInstanceDetailsWholeViewImpl.java]: Why is this class called WholeView? We should have an Advanced and Basic version of the screen right?

File: jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/log/NodeHistoryPopup.java [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+    @UiField
+    public FlowPanel listContainer;
+
+    @Inject
+    private Event<NotificationEvent> notification;
+
+    @Inject
+    private Caller<TaskAuditService> taskAuditService;
+
+    private ListDataProvider<TaskEventSummary> dataProvider = new ListDataProvider<TaskEventSummary>();
```
Comment (Salaboy) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/log/NodeHistoryPopup.java]: We have being working a lot trying to unify the usages of the PagedTable class provided by uberfire-extensions. We should use that in all the places where a data grid is needed.

**Comment by bxf12315** [comment]
@Salaboy  hi i  updated this PR, as we discuss content.
Please review it.

**Comment by bxf12315** [comment]
@Salaboy hi i updated this PR, as we discuss content.(include: refacing Dataset Process instance list)
The PR dependency on PR https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/64 
Please review it.

**Comment by bxf12315** [comment]
replace by new PR

**Comment by bxf12315** [comment]
replace by new PR

**Pull request closed without merging**